include/RAJA/exec-sequential/reduce_sequential.hxx

```diff
@@ -79,30 +79,24 @@ namespace RAJA
 template <typename T>
 class ReduceMin<seq_reduce, T>
 {
+  using my_type = ReduceMin<seq_reduce, T>;
```

Comment:
If you define this type alias here, should you use it everywhere you can; e.g., copy-ctor arg, etc.?